Plugins/Shared/LicenseVerifier.cs

```diff
@@ -316,7 +316,7 @@ private enum FeatureState{
             Guid appId = new Guid(appStr);
 
             XmlDocument doc = new XmlDocument();
-            var root = doc.CreateElement("licenseReqeust");
+            var root = doc.CreateElement("licenseRequest");
```

Comment:
This could be an API change if something external interacts w/ it....

Reply:
Change it. The whole file is unused, not sure why it's still there.